Car_Collision_Visulaization/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 3.0.0)
+project(DataVisualization VERSION 0.1.0 LANGUAGES C CXX)
+
+
+add_library(
+    visualizer
+    src/CImg.h 
+    src/data_visualization.h
+    src/data_visualization.cpp
+    src/car_collision.h
+    src/car_collision.cpp
+)
```

Comment:
1) Library names should start with capital letters
2) Visualizer library should only contain visualization code (car_collision.h and car_collision.cpp) should not be included as they are not part of the same functionality

So Visulization library should look like
add_library(
    Visualizer
    src/CImg.h 
    src/visualizer.h
    src/visualizer.cpp
    )